spyder/plugins/editor/widgets/editorstack/editorstack.py

```diff
@@ -1295,6 +1333,10 @@ def get_current_language(self):
         if self.data:
             return self.data[self.get_stack_index()].editor.language
 
+    def get_current_project_path(self):
+        if self.data:
+            return self.data[self.get_stack_index()].editor.current_project_path
```

Comment:
```suggestion
            finfo = self.get_current_finfo()
            if finfo:
                return finfo.editor.current_project_path
```

I think this is a bit clearer.